pkg/cloudprovider/openstack/openstack.go

```diff
@@ -515,7 +515,7 @@ func (lb *LoadBalancer) CreateTCPLoadBalancer(name, region string, externalIP ne
 
 		_, err = members.Create(lb.network, members.CreateOpts{
 			PoolID:       pool.ID,
-			ProtocolPort: port,
+			ProtocolPort: ports[0], //FIXME: need to handle multi-port
```

Comment:
GCE only supports a range.  It's not ideal, but it works.  If I were implementing I would do both.  e.g. "80,443,8000-9000" :)